Animal.java

```diff
@@ -0,0 +1,52 @@
+ class Animal {
+     String name;
+     int eyes;
+     int eat;
+     int drink;
+     void aminal(){
+         System.out.println("I am animal");
+     }
+     void eat(){
+         System.out.println("Animal eats");
+     }
+
+     void drink(){
+         System.out.println("Animal drinks");
+     }
+     static class Pet {
+         void pet(){
+             System.out.println("I am pet");
+             System.out.println("Pet runs");
+             System.out.println("Pet runs« jumps");
+         }
+         String name;
+         int tai = 1;
+         int paw = 4;
+         String run;
+         String jump;
+
+         class Dog {
+             int paw = 4;
+             void dog(){
+                 System.out.println("I am dog and my name is: Lord + Elli");
+                 System.out.println("Dog plays");
+             }
+         }
+         class Cat {
+             String sleep;
+             void cat(){
+                 System.out.println("I am cat and my name is: Evi + Eva");
+                 System.out.println("Cat sleeps");
+             }
+         }
+     }
+
+
```

Comment:
it's needed to create a separate class Test 

public class Test {
    public static void main(String[] args) {

        Dog dog = new Dog("Buddy");
        Cat cat = new Cat("Whiskers");
        System.out.println("Number of paws: " + dog.paw);
        cat.sleep();
    }
}